discovery-provider/plugins/network-monitoring/src/content/index.ts

```diff
@@ -345,6 +345,11 @@ const _getUserClockValues = async (
 }> => {
 
     try {
+
+        if (endpoint === '') {
```

Comment:
safer for falsey check `if (!endpoint)`? in case it equals `null` instead of `''`

Reply:
why assume? can check. this check might be fine

Reply:
I had checked with the console on audius.co and it was always an empty string. Plus the types for the value that AudiusLibs function say it will always be a string

Reply:
dope - makes sense! sg